framework/controller/loader.go

```diff
@@ -264,7 +266,7 @@ func (l *loader) loadView(controllerKey, actionKey, actionRoute string) *View {
 		if base != key {
 			continue
 		}
-		l.imports.Add(l.module.Import("bud/.app/view"))
+		l.imports.Add(l.module.Import("bud/internal/app/view"))
```

Comment:
I thought `.app` was pretty clever since it the dot causes `goimports` to ignore it, but you can still directly import it. 

This did have the unfortunate side effect that `go mod tidy` would remove dependencies that were only present in `.app/*` because it skipped over them. To counteract this, I added a generated `bud/imports.go` that would link into `.app/*`

Going with `internal/` should simplify everything. They aren't `goimport`ed because they're internal to `bud/`, `go mod tidy` should still pick them up because it's within the module in a standard directory. This also allowed us to remove the `bud/imports.go`.